tools/esp8266/app.cpp

```diff
@@ -250,7 +250,7 @@ void app::loop(void) {
                         DPRINTLN(DBG_DEBUG, F("app:loop WiFi WiFi.status ") + String(WiFi.status()));
                         DPRINTLN(DBG_INFO, F("Requesting Inverter SN ") + String(iv->serial.u64, HEX));
                     }
-                    if(iv->devControlRequest && (iv->powerLimit[0] > 0) && (NoPowerLimit != iv->powerLimit[1])) { // prevent to "switch off"
+                    if(iv->devControlRequest && ((iv->devControlCmd != ActivePowerContr) || ((iv->devControlCmd == ActivePowerContr) && (iv->powerLimit[0] > 0) && (iv->powerLimit[1] != NoPowerLimit)))) { // prevent to "switch off"
```

Comment:
I think this may fix some issue with NoPowerLimit we have seen earlier.

Reply:
Habe ich auch kurz nochmal drüber nachgedacht, aber nein:
Bei ActivePowerContr wird nachwievor kontrolliert, das der neue Wert größer 0 ist und die Einstellung nicht auf NoPowerLimit stehen darf.
ABER das if war vorher so falsch, dass es alle anderen devControlCmd verhindert wurden, wenn die Einstellung auf NoPowerLimit stand, das habe ich hiermit gefixt.

Reply:
Ok, sounds like here that line is fine?